solr/bin/solr

```diff
@@ -148,7 +148,7 @@ fi
 : "${SOLR_START_WAIT:=$SOLR_STOP_WAIT}" # defaulting to $SOLR_STOP_WAIT for backwards compatibility reasons
 
 # Store whether a solr port was explicitly provided, for the "solr stop" command.
-PROVIDED_SOLR_PORT="${SOLR_PORT:-}"
+PROVIDED_SOLR_PORT_ENV="${SOLR_PORT:-}"
```

Comment:
is `_ENV` a pattern we use elsewhere in `bin/solr`?   For these variables?  If we have similar concepts, maybe we can get all of the variables following this pattern?